kvbc/src/pruning_handler.cpp

```diff
@@ -82,6 +82,11 @@ bool RSAPruningVerifier::verify(const concord::messages::PruneRequest& request)
     return false;
   }
 
+  // Make sure pruning parameters are in range.
+  if (request.tick_period_seconds <= 0 || request.batch_blocks_num <= 0) {
```

Comment:
Nit: If I am not mistaken, both `tick_period_seconds` and `batch_blocks_num` are unsigned, I suggest using `==`.